scripts/obtain/dotnet-install.sh

```diff
@@ -115,6 +115,24 @@ get_current_os_name() {
     if [ "$uname" = "Darwin" ]; then
         echo "osx"
         return 0
+    else
+        if [ "$uname" = "Linux" ]; then
+            echo "linux"
+            return 0
+        fi
+    fi
+    
+    say_err "OS name could not be detected: $ID.$VERSION_ID"
+    return 1
+}
+
+get_distro_specific_os_name() {
```

Comment:
That is all good going forward, however, we are effectively disabling any installs of older channels like 'rel-1.1.0':
https://dotnetcli.blob.core.windows.net/dotnet/Sdk/rel-1.1.0/